data/json/professions.json

```diff
@@ -2719,7 +2719,7 @@
     "description": "You spent much of your time prior to the apocalypse at the local mosque, studying the words of the Prophet and the Quran and guiding your community in prayer.  Back then they came from far and wide to listen to you; now they come to eat your brains.",
     "points": 0,
     "//": "No knife, fire, or decent storage/armor.  Skill points are countered.",
-    "skills": [ { "level": 2, "name": "speech" }, { "level": 1, "name": "barter" } ],
+    "skills": [ { "level": 2, "name": "speech" } ],
```

Comment:
Different religious professions were previously "balanced" against one another with the general rule that bartering+speaking is the same for them all. By simply removing bartering without increasing speaking to compensate, you make some of them strictly better than others, which can be seen as somewhat insensitive.
I suggest making them all speaking: 3 to match the defaul "priest" profession.

Reply:
Noted, although it may get tricky when things like persuade and barter become proficiencies